tests/ftqc/test_pauli_tracker.py

```diff
@@ -17,12 +17,23 @@
 
 import numpy as np
 import pytest
+from flaky import flaky
 
 import pennylane as qml
+from pennylane.ftqc import (
+    RotXZX,
+    convert_to_mbqc_formalism,
+    convert_to_mbqc_gateset,
+    diagonalize_mcms,
+    get_byproduct_corrections,
+)
 from pennylane.ftqc.pauli_tracker import commute_clifford_op, pauli_prod, pauli_to_xz, xz_to_pauli
 
 _PAULIS = (qml.I, qml.X, qml.Y, qml.Z)
 
+RTOL = 2.5e-1
+ATOL = 5e-2
```

Comment:
Oooof, this is rough.

Do we need executions here? I think it should be possible to validate the methods work without a full on device execution.

Reply:
Sure!